crates/sncast/tests/e2e/declare.rs

```diff
@@ -127,13 +127,13 @@ async fn test_happy_case_versions(version: &str) {
     assert!(matches!(receipt, Declare(_)));
 }
 
-#[test_case(Some("99999999999999999"), None, None; "max_fee")]
-#[test_case(None, Some("9999"), None; "max_gas")]
-#[test_case(None, None, Some("999999999999"); "max_gas_unit_price")]
+#[test_case(Some("100000000000000000"), None, None; "max_fee")]
+#[test_case(None, Some("100000"), None; "max_gas")]
+#[test_case(None, None, Some("100000000000000"); "max_gas_unit_price")]
 #[test_case(None, None, None; "none")]
-#[test_case(Some("99999999999999999"), None, Some("999999999999"); "max_fee_max_gas_unit_price")]
-#[test_case(None, Some("9999"), Some("999999999999"); "max_gas_max_gas_unit_price")]
-#[test_case(Some("999999999999999"), Some("9999"), None; "max_fee_max_gas")]
+#[test_case(Some("10000000000000000000"), None, Some("100000000000000"); "max_fee_max_gas_unit_price")]
+#[test_case(None, Some("100000"), Some("100000000000000"); "max_gas_max_gas_unit_price")]
+#[test_case(Some("100000000000000000"), Some("100000"), None; "max_fee_max_gas")]
```

Comment:
Why were these changed?

Reply:
@integraledelebesgue

Reply:
When the prices changed, these tests became invalid. Also, it's more convenient to operate on powers of ten